scripts/gather_feature_toggle_state.py

```diff
@@ -127,6 +127,21 @@ def build_query(table_name):
             "LEFT JOIN waffle_flag_groups ON waffle_flag.id=waffle_flag_groups.flag_id "
             "GROUP BY waffle_flag_users.flag_id, waffle_flag_groups.flag_id;"
         )
+    elif table_name == 'waffle_utils_waffleflagcourseoverridemodel':
+        # This model is keyed on waffle_flag and course_id, and the
+        # record with the newest change_date for a given key is the
+        # one in effect (this is the contract of ConfigurationModel).
+        current_versions_subquery = (
+            "SELECT waffle_flag, course_id, MAX(change_date) AS max_change_date "
```

Comment:
Consider `max_change_date` => `date_modified`

Reply:
Hmm. Come to think of it, I actually don't need to alias it at all, since I don't select or join it by name -- I could just change this line to `SELECT waffle_flag, course_id, MAX(change_date)`.